earthkit/data/readers/csv.py

```diff
@@ -134,13 +134,26 @@ def __init__(self, source, path, compression=None):
         self.compression = compression
         self.dialect, self.has_header = probe_csv(path, compression=compression)
 
-    def to_pandas(self, pandas_read_csv_kwargs=None):
+    def to_pandas(
+        self, comment="#", compression=None, pandas_read_csv_kwargs=None, **kwargs
```

Comment:
Why do we need **kwargs here?

Reply:
This was to make the user API nicer such that if we are calling to pandas on a CSV reader you don't need to nest the kwargs, they can be passed in directly. The nesting is maintained such that when to_pandas is called by an intermediate method, the read_csv_kwargs can be parsed without interfering with intermediate method. Hopefully that makes sense, we can discuss in person if you want

Reply:
As discussed the `kwargs` are not used by this method. However, I have left them in the definition such that the code does not break if a user provides kwargs that are not applicable, e.g. the ODB kwargs.